### ISSUE TITLE
Beta showing duplicate videos on subscription page

### ISSUE THREAD

**Issue by 42degrees** [lbryio/lbry-app#1696]
In the most recent betas, I am seeing every video on my subscription page duplicated.  Here is a screen shot.

![image](https://user-images.githubusercontent.com/1676142/41925195-209dade4-7929-11e8-8008-e165de956775.png)

Every one that is duplicated is from @timcast.  Clicking on the duplicates takes me to the identical video for each duplicate.  I subscribed to other channels and the videos on those other channels are not duplicated at this time (but @timcast wasn't duplicated when I first subscribed to it either).

![image](https://user-images.githubusercontent.com/1676142/41925370-a34342d6-7929-11e8-9352-377ce913ff47.png)

**Comment by tzarebczan** [comment]
Thanks for opening this @42degrees. Just checked our api and it's stored twice there (not supposed to happen). The fix may be on that side of the house, will take a closer look. I'm leaving as such for now so we can debug :)

**Comment by tzarebczan** [comment]
yep, an issue on our API side, the record was stored twice (not supposed to happen). Only happened to one other person, but I'll leave this issue open for visibility in case someone else runs into it.

**Comment by 42degrees** [comment]
Confirmed, LBRY 0.23.0 (or daemon 0.20.4) fixed the duplication in my subscriptions page.  I did not need to clear my cache.
